src/components/moj-timeline/README.md

```diff
@@ -3,7 +3,81 @@
 - [Guidance](https://mojdt-design-system.herokuapp.com/components/timeline)
 - [Preview](https://mojdt-frontend.herokuapp.com/components/timeline)
 
+## Dependencies
```

Comment:
I don't think we need to specify any of this here. The dependency should be included in the package

Reply:
So in the `package/package.json` should include the following?
```
"dependencies": {
  "moment": "^2.22.2"
}
```